CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Allow define typescript project with `-p` parameter. This is useful to have a `tsconfig.json` in the root to allow IDE and library implementation to get a project file including files even if they won't be included in the NPM package (.spec, .test, .stories). Otherwise a project including this files will transform every file defined, even those not desired in the final bundle, not only forcing to exclude them explicitly but increasing compilation times and errors. DefaiBu default `tsconfig.json` will be used as it usually does.
```

Comment:
Fixing typos and including some information about the `-p` parameter:

```suggestion
- Allow defining typescript project with `-p` parameter to define a different `tsconfig.ts` file. p.eg: `package-build -p tsconfig.build.json`. This is useful to have a `tsconfig.json` in the root to allow IDE and library implementation to get a project file including files even if they won't be included in the NPM package (.spec, .test, .stories). Otherwise, a project including these files will transform every file defined, even those not desired in the final bundle, not only forcing to exclude them explicitly but increasing compilation times and errors. By default `tsconfig.json` will be used as it usually does.
```